src/viur/core/request.py

```diff
@@ -76,7 +74,7 @@ def validate(request: 'BrowseHandler') -> typing.Optional[typing.Tuple[int, str,
         return 403, "Forbidden", "Request rejected due to fetch metadata"
 
 
-class BrowseHandler():  # webapp.RequestHandler
+class Request:
```

Comment:
I would call it `Router`, otherwise you've nested the `Request` of the `viur-core` and the `Request` of webob.
```suggestion
class Router:
```

Reply:
But the context variable is called `current.request`, therefore I chose this name. Nevermind, I want to finalize rc1 now.